packages/odata-common/src/de-serializers/default-de-serializers.ts

```diff
@@ -108,7 +108,17 @@ export const defaultDeSerializersRaw: DefaultDeSerializers = {
   /* DeSerializers with v2 and v4 specific URI serializer defaults. */
   'Edm.Decimal': {
     deserialize: deserializeToBigNumber,
-    serialize: serializeFromBigNumber
+    serialize: value => {
+      const primitiveNumber =
+        value instanceof BigNumber ? value.toNumber() : value;
```

Comment:
[req] We had issues with `instanceof` in the past when you have multiple "instances" of the class `BigNumber` in your `node_modules`. Therefore I would just turn it around.
```suggestion
      const primitiveNumber =
        typeof value === 'number' ? value : value.toNumber();
```